.travis.yml

```diff
@@ -121,6 +121,10 @@ script:
   - echo $MAIN_CMD $SETUP_CMD
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then pip install pytest; fi
   - eval $MAIN_CMD $SETUP_CMD
+  # check benchmark suite for basic issues
```

Comment:
Would it be useful to make `asv check` a separate job and just run it on whatever we are actually running it on (namely Python 2.7...) instead of running it for every job?

I know that it's only a few seconds but the diagnostics would be cleaner if, say, only the asv check job fails while everything else still passes.